### PR TITLE
Add Ukrainian language

### PR DESCRIPTION
**Pull request by samalero**
Description
-
add Ukrainian language and fix English locale file

Reason to modify
-
For example, for using on my server

Tests & Reviews
-

### CODE REVIEW

**Comment by KairusDarkSeeker** [comment]
Follow the PR format of else this will not get merged

**Comment by samalero** [comment]
done

File: src/pocketmine/lang/locale/eng.ini [hunk 1]
```diff
@@ -213,13 +223,14 @@ commands.xp.failure.withdrawXp=Cannot give player negative experience points
 commands.xp.success=Given {%0} experience to {%1}
 commands.xp.success.levels=Given {%0} levels to {%1}
 commands.xp.success.negative.levels=Taken {%0} levels from {%1}
-commands.xp.usage=/xp <amount> [player] OR /xp <amount>L [player]
+commands.xp.usage=/xp <amount> [player] OR /xp <amount of levels> [player]
```
Comment (Tarik02) [inline src/pocketmine/lang/locale/eng.ini]: `OR /xp <amount of levels> [player]` - why you removed `L`? This must be a `/xp <amount of levels>L [player]`.

**Comment by Tarik02** [comment]
Also about translation, I am ukrainian, it's right.

File: src/pocketmine/lang/locale/eng.ini (hunk 1 above)
Comment (samalero) [inline src/pocketmine/lang/locale/eng.ini]: Ok, i'm back this.

**Comment by samalero** [comment]
І що з того?! Не подобається переклад - вказуй на помилки...

File: src/pocketmine/lang/locale/eng.ini [hunk 2]
```diff
@@ -168,7 +175,7 @@ commands.save.start=Saving...
 commands.save.success=Saved the world
 
 commands.setblock.usage=/setblock <x> <y> <z> <BlockName> [damage]
-command.setblock.invalidBlock=Invalid block name/ID
+commands.setblock.invalidBlock=Invalid block name/ID
```
Comment (iPocket) [inline src/pocketmine/lang/locale/eng.ini]: You sure?

**Comment by samalero** [comment]
what about this? i'm added this parameters and i'm verified this.

```
pocketmine.command.fill.missingParameter=Missing parameters
pocketmine.command.fill.usage=/fill <xMin> <yMin> <zMin> <xMax> <yMax> <zMax> <ID>
```

File: src/pocketmine/lang/locale/eng.ini (hunk 2 above)
Comment (samalero) [inline src/pocketmine/lang/locale/eng.ini]: sorry i'm revert this

**Comment by ishitatsuyuki** [comment]
One thing per commit. Rebase and squash the redundant fix.

**Comment by KairusDarkSeeker** [comment]
😐

**Comment by ishitatsuyuki** [comment]
Do not attempt to spam PRs; you will directly get a ban for wasting our time.

**Comment by samalero** [comment]
Ok ok please reopen pull request and i revert change in eng.ini

**Comment by iPocket** [comment]
I can't reopen this due to you have deleted your branch.

**Pull request closed without merging**